packages/core/src/2d/sprite/SpriteRenderer.ts

```diff
@@ -339,7 +343,7 @@ export class SpriteRenderer extends Renderer {
 
     // Update position
     if (this._dirtyUpdateFlag & RendererUpdateFlags.WorldVolume) {
-      this._assembler.updatePositions(this);
+      this._assembler.updatePositions(this, this.width, this.height, sprite.pivot);
```

Comment:
Here’s an example of how you could simplify it:
const { width, height, _assembler } = this;
_assembler.updatePositions(this, width, height, sprite.pivot);